helpers/purchaseTypes.js

```diff
@@ -0,0 +1,18 @@
+export const PURCHASE_TYPES = {
+  DOMAIN_REGISTRATION: 'DOMAIN_REGISTRATION',
+  PRO_SUBSCRIPTION: 'PRO_SUBSCRIPTION',
+  ADDITIONAL_FEATURES: 'ADDITIONAL_FEATURES',
```

Comment:
Seems like we should remove these until we actually add these other methods of purchase, could lead to confusion...